Code/GraphMol/AddHs.cpp

```diff
@@ -319,7 +319,8 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
 }  // end of unnamed namespace
 
 namespace MolOps {
-void addHs(RWMol &mol, bool explicitOnly, bool addCoords) {
+void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
+           const UINT_VECT *onlyOnAtoms) {
```

Comment:
It might also make sense to do the same for the onlyOnAtoms

```
void addHs(RWMol &mol, const UINT_VEC * onlyOnAtoms, bool explicitOnly, bool addCoords);
```

otherwise the user would need to know the defaults for explicitOnly, addCoords.  This could still call the version you wrote but perhaps be a better API where the Query version could be used in the future.